src/app/presentation/components/home-components/about/about.component.html

```diff
@@ -31,9 +31,9 @@ <h1 class="registration_title">About Us</h1>
               <mat-grid-tile>
                   <div class="about-text text-left justify-start">
                       <h3 class="feature-title">About team</h3>
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore.</p>
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                      <p>We are the young team, we are from different cities, we have different education, but we are united by one goal – to become the great developers.</p>
+                      <p>We have known each other in all for three month, but despite this we are choosing, planning, and starting the development of our project. Many of using technologies and frameworks was new for us, and we spend lot of time on the beginning to investigate it. And additionally nobody worked with SCRUM before. But we have done a great job, and you will see that from first minutes in our application</p>
```

Comment:
Replace "in all" with "only", "month" with "months"
Many of **_used_** technologies and frameworks was new for us, and we spen**_t_**  **_a_** lot of time **_at_** the beginning to investigate it.
**_Also_** nobody **_has_** worked with SCRUM before. **_However_**, we have done great job. You'll see **_it_** from first minutes in our application.
Split this paragraph into few lines, HTML allows that.